lib/Models/registerCatalogMembers.js

```diff
@@ -177,11 +177,21 @@ var registerCatalogMembers = function() {
     WebFeatureServiceCatalogGroup,
     true
   );
+  createCatalogItemFromUrl.register(
```

Comment:
Rather than registering the item again could we do improve the regex on the first registration so it captures both options, eg `(\/wfs|=wfs)` - that seems better than registering the item twice...